shotover-proxy/src/transforms/cassandra/peers_rewrite.rs

```diff
@@ -162,74 +196,174 @@ mod test {
 
     #[test]
     fn test_is_system_peers_v2() {
-        assert!(is_system_peers(&mut create_query_message(
-            "SELECT * FROM system.peers_v2;".into()
-        )));
+        let peer_table = FQName::new("system", "peers_v2");
+        let v = extract_native_port_column(
+            &peer_table,
+            &mut create_query_message("SELECT * FROM system.peers_v2;"),
+        );
+        assert_eq!(1, v.len());
```

Comment:
These kind of asserts are more robust if we write them like: `assert_eq!(v, vec!("native_port".to_string()));`